xml/System.Text.Json/JsonElement.xml

```diff
@@ -641,10 +704,12 @@
         <Parameter Name="value" Type="System.Single" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">Receives the value.</param>
+        <summary>Attempts to represent the current JSON number as a <see cref="T:System.Single" />.</summary>
+        <returns><see langword="true" /> if the number can be represented as a <see cref="T:System.Single" />, <see langword="false" /> otherwise.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.InvalidOperationException"> This value's <see cref="P:System.Text.Json.JsonElement.Type" /> is not <see cref="F:System.Text.Json.JsonValueType.Number" />.</exception>
```

Comment:
nit: For example another place where we have a space (look for `"> `).
```suggestion
        <exception cref="T:System.InvalidOperationException">This value's <see cref="P:System.Text.Json.JsonElement.Type" /> is not <see cref="F:System.Text.Json.JsonValueType.Number" />.</exception>
```

Reply:
```suggestion
        <exception cref="T:System.InvalidOperationException">This value's <see cref="P:System.Text.Json.JsonElement.Type" /> is not <see cref="F:System.Text.Json.JsonValueType.Number" />.</exception>
```